### PR TITLE
LEETCODE-1022 Added Sort List

### PR DESCRIPTION
**Pull request by Vasis3038**

### CODE REVIEW

File: leetcode/linked-list.md [hunk 1]
```diff
@@ -0,0 +1,313 @@
+# linked-list
```
Comment (alexarlord-boop) [inline leetcode/linked-list.md:1]: # Invalid PULL Commits

** Invalid Commit Message: LEETCODE added Reverse Linked List **
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

** Invalid Commit Message: LEETCODE added Middle of the Linked List **
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

** Invalid Commit Message: LEETCODE added Palindrome Linked List **
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

** Invalid Commit Message: LEETCODE-1022 Merge Two Sorted Lists **
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

** Invalid Commit Message: LEETCODE-1022 Remove Nth Node From End of List **
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

Comment (alexarlord-boop) [inline leetcode/linked-list.md:1]: # Invalid PULL Commits

** Invalid Commit Message: LEETCODE added Reverse Linked List **
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

** Invalid Commit Message: LEETCODE added Middle of the Linked List **
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

** Invalid Commit Message: LEETCODE added Palindrome Linked List **
! Message must contain group number in ['1021', '1022']
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

** Invalid Commit Message: LEETCODE-1022 Merge Two Sorted Lists **
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

** Invalid Commit Message: LEETCODE-1022 Remove Nth Node From End of List **
! Message must start with ['Added', 'Deleted', 'Fixed', 'Refactored', 'Moved']

Comment (Vasis3038) [inline leetcode/linked-list.md:1]: ALL WRONG in commit message/n/nALL WRONG in commit message/n/nALL WRONG in commit message

Comment (Vasis3038) [inline leetcode/linked-list.md:1]: ALL WRONG in commit message/nALL WRONG in commit message/nALL WRONG in commit message

Comment (Vasis3038) [inline leetcode/linked-list.md:1]: ALL WRONG in commit message

ALL WRONG in commit message

ALL WRONG in commit message

**Pull request merged**